timestampvm/vm.go

```diff
@@ -258,3 +259,24 @@ func (vm *VM) Connected(id ids.ShortID) error {
 func (vm *VM) Disconnected(id ids.ShortID) error {
 	return nil // noop
 }
+
+
```

Comment:
can we also put the same comments here like you did below?

Reply:
Ok added